lib/charms/mongodb/v0/helpers.py

```diff
@@ -64,7 +66,7 @@ def get_create_user_cmd(config: MongoDBConfiguration, mongo_path="mongo") -> Lis
     ]
 
 
-def get_mongod_cmd(config: MongoDBConfiguration) -> str:
+def get_mongod_cmd(config: MongoDBConfiguration, mongo_files_dir="/etc/mongodb") -> str:
```

Comment:
Is this path correct?  Shouldn't this be under common?

Reply:
that is the default directory, it defaults to `etc/mongodb` so that the k8s charm won't require any changes when the lib gets updated